src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -475,6 +540,136 @@ const ManifestComponent = ({
+        if (isConfigDriftEnabled && appDetails?.appType === AppType.DEVTRON_APP && showManifestCompareView) {
+            return normalizedLiveManifest
+        }
+
+        return trimedManifestEditorData
+    }
+
+    const renderEditorInfo = (isCodeEditorView: boolean = false) => {
+        if (!showInfoText) {
+            return null
+        }
+
+        const message =
+            isEditMode && !showManifestCompareView
+                ? EA_MANIFEST_SECRET_EDIT_MODE_INFO_TEXT
+                : EA_MANIFEST_SECRET_INFO_TEXT
+
+        if (isCodeEditorView) {
+            return (
+                <CodeEditor.Information text={message} className="flex left">
+                    {renderShowDecodedValueCheckbox()}
+                </CodeEditor.Information>
+            )
+        }
+
+        return (
+            <InfoColourBar
+                message={message}
+                classname="w-100 m-0 code-editor__information dc__no-border-radius dc__no-top-border dc__no-left-border dc__no-right-border dc__word-break"
+                Icon={ICInfoFilled}
+                iconClass="icon-dim-16"
+                linkClass="dc__truncate--clamp-6"
+            />
+        )
+    }
+
+    const renderErrorBar = (isCodeEditorView: boolean = false) => {
+        if (showManifestCompareView || !errorText) {
+            return null
+        }
+
+        if (isCodeEditorView) {
+            return <CodeEditor.ErrorBar text={errorText} />
+        }
+
+        return (
+            <InfoColourBar
+                message={errorText}
+                classname="w-100 m-0 code-editor__error dc__no-border-radius dc__no-top-border dc__no-left-border dc__no-right-border dc__word-break"
+                Icon={ICErrorExclamation}
+                iconClass="icon-dim-16"
+                linkClass="dc__truncate--clamp-6"
+            />
+        )
+    }
+
+    const renderContent = () => {
+        if (!isReadOnlyView && manifestFormConfigurationType === ConfigurationType.GUI) {
+            return (
+                <>
+                    {renderEditorInfo()}
+                    {renderErrorBar()}
+                    <ManifestGUIView
+                        guiSchema={guiSchema}
+                        handleChange={handleGUIViewValueChange}
+                        // For uniformity have called method but as of now in this case it will always be trimedManifestEditorData
+                        manifestYAMLString={getCodeEditorValue()}
+                        handleSwitchToYAMLMode={handleSwitchToYAMLMode}
+                    />
+                </>
+            )
+        }
+
+        return (
+            <CodeEditor
+                defaultValue={showManifestCompareView && desiredManifest}
+                cleanData={showManifestCompareView}
+                diffView={showManifestCompareView}
+                theme="vs-dark--dt"
+                height={isResourceBrowserView ? 'calc(100vh - 119px)' : 'calc(100vh - 77px)'}
```

Comment:
suggestion: use flex based height approach

Reply:
We should as a part of part of refactoring since had only moved this code block above and parent divs are also calculated which might have major impact